libyul/AsmAnalysis.cpp

```diff
@@ -263,6 +263,8 @@ vector<YulString> AsmAnalyzer::operator()(FunctionCall const& _funCall)
 		returnTypes = &f->returns;
 		if (f->literalArguments)
 			needsLiteralArguments = &f->literalArguments.value();
+
+		warnOnInstructions(_funCall.functionName.name.str(), _funCall.functionName.location);
```

Comment:
Won't this be also executed during `--strict-assembly`

Reply:
I think it should be executed during `--strict-assembly` too (according to https://github.com/ethereum/solidity/issues/7965) Then the changelog needs to be updated.

Reply:
This will also be executed during `--strict-assembly` mode, yes. I think this is expected, isn't it?